test/libsolidity/syntaxTests/natspec/invalid/invalid_tag.sol

```diff
@@ -14,6 +14,6 @@ contract C {
 // ----
 // DocstringParsingError 6546: (0-14): Documentation tag @a&b not valid for contracts.
 // DocstringParsingError 2968: (28-49): Invalid character in custom tag @custom:x^y. Only lowercase letters and "-" are permitted.
-// DocstringParsingError 6546: (80-92): Documentation tag @custom: not valid for functions.
+// DocstringParsingError 6564: (80-92): Custom documentation tag must contain a chosen name, i.e. @custom:mytag.
```

Comment:
I swear I used the error code script to generate the number, but somehow RNGesus decided 6546 -> 6564 is a good number.